packages/launchpad/cypress/tasks/snapshotsScaffold.ts

```diff
@@ -5,7 +5,7 @@ import disparity from 'disparity'
 import dedent from 'dedent'
 import { cyTmpDir } from '@tooling/system-tests/lib/fixtures'
 
-const systemTestsDir = path.join(__dirname, '..', '..', '..', '..', 'system-tests', 'projects')
+const systemTestsDir = path.posix.join(__dirname, '..', '..', '..', '..', 'system-tests', 'projects')
```

Comment:
on Windows, `__dirname` will have backslashes in it, and path.posix.join will not do what is expected - it will treat the entire `__dirname` as a single filename: 
![image](https://user-images.githubusercontent.com/1151760/161153937-0c6d08aa-f1bc-46bd-9561-ff1b1c888572.png)

I'd just keep this as a native `path.join` and convert to `posix` paths at only the places necessary.

Reply:
Right, I see - thanks.

Reply:
Ok, I think I've done this right now. We only use posix for the globby globs now. Can you please double check? Thanks.